crates/transaction-emitter-lib/src/emitter/submission_worker.rs

```diff
@@ -341,16 +382,21 @@ pub async fn submit_transactions(
                         } else {
                             None
                         };
+                    let balance = client
+                        .get_account_balance(sender)
```

Comment:
this is extra details on sample of failed transactions to understand why.

for example if failed transactions are because of low funds, to be able to see the balance itself, and why it was so low.
similarly how we fetch last successful txn, to be able to debug issues with sequence numbers, for example.